subsys/bluetooth/mesh/prov_device.c

```diff
@@ -742,7 +742,7 @@ int bt_mesh_prov_enable(bt_mesh_prov_bearer_t bearers)
 		return -EALREADY;
 	}
 
-	if (IS_ENABLED(CONFIG_BT_DEBUG)) {
+	if (IS_ENABLED(CONFIG_BT_MESH_PROV_DEVICE_LOG_LEVEL_DBG)) {
```

Comment:
Could you please replace this with `CONFIG_BT_MESH_PROV_DEVICE_LOG_LEVEL_INF`? The log message below is supposed to be printed with Info level.

Reply:
Hmm, does the same apply to `hci_vs_init` above? We guard `LOG_INF` with the debug level.

Reply:
You are right I think, I will update with `INF` everywhere. All are used to guard `LOG_INF`.